README.md

```diff
@@ -35,6 +37,19 @@ OR, run container with config file overriding defaults
 docker run -p 4500:4500 ga4gh/ga4gh-starter-kit-drs:latest java -jar ga4gh-starter-kit-drs.jar -c path/to/config.yml
 ```
 
+#### PostgreSQL
+
+Postgres will be run on a separate container and simultaneously ran with Starter Kit DRS via `docker compose`:
+
+Clone this repo:
+```
+git clone https://github.com/ga4gh/ga4gh-starter-kit-drs.git
+```
+
+Navigate into the repo's root folder
+
+docker compose -f docker-compose-postgres.yml up -d
+
```

Comment:
Could you please include the expected outcomes in your explanation? For example, when running docker ps, which containers should be listed? What port is the PostgreSQL database available on, and can you briefly describe the connection process between the DRS Starter Kit Docker image and the PostgreSQL database using the configuration file? It would be helpful if you could mention which specific configuration fields are utilized to establish the connection with the PostgreSQL database.

Reply:
Please add information on how to shutdown the services that were created by running the above command.